### DISCUSSION TITLE
Adaptive Steuerung und analoges Gamepad

### DISCUSSION THREAD

**Discussion by Gecko774** [Mogli12/KeyboardSteer#735] [Ideas]
Hallo,

die Adaptive Steuerung funktioniert leider nicht mit einem Gamepad.

Ist das so erwünscht oder liegt da noch ein Fehler vor?

Falls es so angedacht ist, wäre es denn bitte auch möglich diese Steuerung für Gamepads (Xbox One etc.) einzubauen?
Kann mich erinnern, dass dies im LS19 anfangs noch ging, bzw. mit einer der Versionen die ich benutzte.

Trotzdem vielen Dank für VCA 👍

**Comment by Mogli12** [comment]
Hallo @Gecko774,

probiere mal die Version 102. Dort kannst Du die analogen Controller in den globalen Einstellungen von VCA einschalten.

Viele Grüße

**Reply by Gecko774** [reply]
Vielen Dank der Controller wird erkannt. :hugs:
Einzig die einzelnen Einstellung dann mit der Lenkgeschwindigkeit und Rückstellung besonders mit niedrigeren Werten (<50%) werden nicht ganz angenommen.
Aber man merkt aufjedenfall auch schon so eine deutliche Verbesserung :+1:

**Comment by Mogli12** [comment]
Es gibt jetzt auch eine 2. Implementierung der adaptiven Lenkung. Das ist eine uralte Implementierung, die etwas träge reagiert dadurch aber zu einer ruhigeren Lenkung führt.

**Comment by Gecko774** [comment]
Vielen Dank nochmals.
VCA mit sämtlichen Einstellungen gehört definitv zu den "Must Have Mods" der LS Reihe 👍
